schematic/schemas/data_model_relationships.py

```diff
@@ -0,0 +1,222 @@
+                    'csv_header': 'Validation Rules',
+                    'node_label': 'validationRules',
+                    'jsonld_direction': 'out',
+                    'edge_dir': 'out',
+                    'jsonld_default': [],
+                    'type': list,
+                    'edge_rel': False,
+                    'required_header': True,
+                    'node_attr_dict':{'default': [],
+                                      'standard': parse_validation_rules,
+                                },
+                },
+                'domainIncludes': {
+                    'jsonld_key': 'schema:domainIncludes',
+                    'csv_header': 'Properties',
+                    'edge_key': 'domainValue',
+                    'jsonld_direction': 'in',
+                    'edge_dir': 'in',
+                    'type': list,
+                    'edge_rel': True,
+                    'required_header': True,
+                },
+                'isPartOf': {
+                    'jsonld_key': 'schema:isPartOf',
+                    'csv_header': None,
+                    'node_label': 'isPartOf',
+                    'type': dict,
+                    'edge_rel': False,
+                    'required_header': False,
+                    'node_attr_dict':{'default': {"@id": "http://schema.biothings.io"},
+                                },
+                },
+                'id': {
+                    'jsonld_key': '@id',
+                    'csv_header': 'Source',
+                    'node_label': 'uri',
+                    'type': str,
+                    'edge_rel': False,
+                    'required_header': True,
+                    'node_attr_dict':{'default': get_label_from_display_name,
+                                    'standard': get_label_from_display_name,
+                                    },
+                },
+            }
+
+        return map_data_model_relationships
+
+    def define_required_csv_headers(self):
+        """Helper function to retrieve required CSV headers, alert if required header was not provided.
+            Returns:
+                required_headers: lst, Required CSV headers.
+        """
+        required_headers = []
+        for k, v in self.relationships_dictionary.items():
+            try:
+                if v['required_header']:
+                    required_headers.append(v['csv_header'])
+            except KeyError:
+                print(f"Did not provide a 'required_header' key, value pair for the nested dictionary {k} : {key}")
+
+        return required_headers
+
+    def define_edge_relationships(self):
+        """Helper function to retrieve CSV headers for edge relationships.
+            Returns:
+                edge_relationships: dict, key: csv_header if the key represents an edge relationship.
+        """
+        edge_relationships = {}
+        for k, v in self.relationships_dictionary.items():
```

Comment:
should we avoid using single character as variable name here? I think `k` could be changed to "relationship_name" and `v` could be changed to `relationship_info`?